[Bug] CALM_DSL_CONFIG_FILE_LOCATION envvar is ignored

**Describe the bug**
CALM_DSL_CONFIG_FILE_LOCATION envvar is ignored. Inline -c config_file works.

**Branch/Container**
master

**To Reproduce**
Steps to reproduce the behavior:
1. export CALM_DSL_CONFIG_FILE_LOCATION=/my/config_file/path
2. make sure you don't have a config.ini file on your $HOME/.calm/config.ini
3. run something like calm get bps
4. See error

```shell
root ➜ /workspaces/calm-community (main ✗) $ calm get bps
[2022-07-24 09:06:30] [ERROR] [calm.dsl.config.context:62] Host IP not found. Please provide it in config file or set environment variable 'CALM_DSL_PC_IP'
[2022-07-24 09:06:30] [ERROR] [calm.dsl.config.context:53] Invalid config file location '/root/.calm/config.ini'
```
**Expected behavior**
It should use the environment variable, and if it doesn't exist, fallback to the default path

**Findings/Troubleshooting Done**
With inline configuration it works

```shell
calm -c config/forrester.ini get bps
```

** System Configuration:**
- OS: container python:3-bullseye
- Python: 3.10.4

--- a/calm/dsl/config/init_config.py
+++ b/calm/dsl/config/init_config.py
@@ -53,21 +53,21 @@ class InitConfigHandle:
             config_obj["CONFIG"] = {"location": DEFAULT_CONFIG_FILE}
 
         if env_init_config.get("config_file_location"):
-            config_obj["CONFIG"]["LOCATION"] = env_init_config["config_file_location"]
+            config_obj["CONFIG"]["location"] = env_init_config["config_file_location"]
 
         if not config_obj.get("DB", {}).get("location"):
             make_file_dir(DEFAULT_DB_LOCATION)
             config_obj["DB"] = {"location": DEFAULT_DB_LOCATION}
 
         if env_init_config.get("db_location"):
-            config_obj["DB"]["LOCATION"] = env_init_config["db_location"]
+            config_obj["DB"]["location"] = env_init_config["db_location"]
 
         if not config_obj.get("LOCAL_DIR", {}).get("location"):
             make_file_dir(DEFAULT_LOCAL_DIR_LOCATION)
             config_obj["LOCAL_DIR"] = {"location": DEFAULT_LOCAL_DIR_LOCATION}
 
         if env_init_config.get("local_dir_location"):
-            config_obj["LOCAL_DIR"]["LOCATION"] = env_init_config["local_dir_location"]
+            config_obj["LOCAL_DIR"]["location"] = env_init_config["local_dir_location"]
 
         self._CONFIG = config_obj
 

